topics/thelounge/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: thelounge-theme
+display_name: The Lounge
+github_url: https://github.com/thelounge/lounge
+logo: thelounge.png
+related: irc, irc-client, chat, javascript
+released: February 12, 2016
+short_description: The Lounge is a modern web IRC client that designed for self-hosting.
```

Comment:
I think you're missing a word: "client that designed" => "client that was designed".

Instead of "for self-hosting", what about "to be self-hosted"?

Reply:
Good catch, I did mess up the sentence, and didn't even catch that when I re-read :man_facepalming: 
Will fix.